src/import.rs

```diff
@@ -0,0 +1,47 @@
+use crate::utils::{clear_wallets_vault, DEFAULT_WALLETS_VAULT_PATH};
+use anyhow::{bail, Result};
+use fuels::signers::wallet::Wallet;
+use std::path::PathBuf;
+
+pub(crate) fn import_wallet(path: Option<String>) -> Result<()> {
+    let wallet_path = match &path {
+        Some(path) => {
+            // If the provided path exists but used clear it
+            clear_wallets_vault(&PathBuf::from(&path))?;
```

Comment:
```suggestion
            clear_wallets_vault(path)?;
```

Reply:
Ahh I see! A nicer approach in this case might be to use `path.as_ref()` as `str` implements `AsRef<Path>`: see [here](https://doc.rust-lang.org/std/primitive.str.html#impl-AsRef%3CPath%3E). This avoids the allocation required for the `PathBuf`.

This is a very minor nit though - happy to land as is!